TransferMethodRepository/Sources/TransferMethodConfigurationRepository.swift

```diff
@@ -48,7 +48,7 @@ public protocol TransferMethodConfigurationRepository {
     func refreshKeys()
 }
 
-// MARK: - RemoteTransferMethodConfigurationRepository
+/// RemoteTransferMethodConfigurationRepository
```

Comment:
Think `MARK` and documentation serve different purposes - please keep the `MARK`